src/adr/0064-account-delegation.md

```diff
@@ -0,0 +1,48 @@
+# 64. Account Delegation
+
+Date: 2022-11-15
+
+## Status
+
+Accepted (lazy consensus).
+
+Not yet implemented.
+
+## Context
+
+The account delegation is a feature that gives another permission to access your account and vice versa.
+The user has been delegated can begin accessing that account from your own account,
```

Comment:
The user that had been granted access  begin accessing that account then read, send, respond to, and delete email messages... on your behalf.